### ISSUE TITLE
[Suggestion] Show search result on home page

### ISSUE THREAD

**Issue by SCDerox** [discordlist-gg/beta-issues#23]
### TL;DR
Show up to 3 (or more idk) search results on the main page when entering something in the search bar.

### Current behavior 
There is a big search bar on the start page (`/`). If I enter something I have to press enter to execute the search.
![Screenshot from 2021-08-28 11-59-06](https://user-images.githubusercontent.com/43812987/131214260-5177aec3-fd1e-4c1d-9bd7-2b7536355a4d.png)

### Suggested behavior
The search should be executed as the user types and up to three bots should be displayed as results. There should also be a big "Show all results" button or similar. If nothing is being searched I suggest to show the top voted bots or nothing.
![Screenshot from 2021-08-28 12-04-11](https://user-images.githubusercontent.com/43812987/131214371-7672333f-dff9-47cf-8279-1bec6f6eb6c2.png)

### Why is this a good idea?
I expect that a lot of new visitors to dlist will use the search page on the home page. To make it easier to find the bots they are looking for, I suggest to implement something like this, so users don't have to press enter (or the search button).

**Comment by audn** [comment]
Thank you for taking the extra step and showing us an example!

I really like this idea, so I went ahead and added the **planned** label.
However, this is something I want to do after the giveaway, as the homepage has to be re-designed due to the giveaway being invalid after it's done.
